src/codeu/chat/client/maingui/LoginPanel.java

```diff
@@ -0,0 +1,72 @@
+package codeu.chat.client.maingui;
+import java.awt.*;
+import java.awt.event.ActionListener;
+import java.awt.event.ActionEvent;
+import javax.swing.*;
+
+import codeu.chat.client.ClientContext;
+import codeu.chat.common.LoginInputCallback;
+
+@SuppressWarnings("serial")
+public final class LoginPanel extends JPanel{
+
+    private final ClientContext clientContext;
+    private final LoginInputCallback loginInputCallback;
+
+    public LoginPanel(ClientContext clientContext, LoginInputCallback loginInputCallback) {
+        super();
+        this.clientContext = clientContext;
+        this.loginInputCallback = loginInputCallback;
+        initialize();
+    }
+    private void initialize() {
+
+        // This panel contains the login form for get access to the chat.
```

Comment:
BTW, yes, a more typical workflow here would be to resolve Yoali's comment first before completing the merge -- create a new commit that addresses review comments in login-main-view and push that back to github, that should automatically update the pull request.  (I think those are the steps based on http://stackoverflow.com/a/7947337, I've never used git+github).

Resolving a comment in a follow up merge sometimes makes sense if, e.g., the solution is big and out of scope of your change ("refactor this method" or something).